Gruntfile.js

```diff
@@ -78,6 +78,15 @@ module.exports = function(grunt) {
           summary: true
         }
       },
+      consoleLoggingOptions: {
+        src: 'test/fixtures/pivotal/src/**/*.js',
+        options: {
+          specs: 'test/fixtures/pivotal/spec/*Spec.js',
+          helpers: 'test/fixtures/pivotal/spec/*Helper.js',
+          display: 'short',
+          showLogging: true
```

Comment:
I wasn't getting the impression that the provided tests actually test the behavior of grunt-contrib-jasmine. Rather, they are varied to demonstrate how running different tests will behave. 

I hear you 100%, but I was just following the existing pattern.

side note: in my original version of this PR, I didn't change any tests -- but I did change the console output behavior -- so I know the console output tests weren't testing console output behavior.